server/trillian_log_server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang
+
+ENV DB_USER=test \
```

Comment:
It might be better to use [ARG](https://docs.docker.com/engine/reference/builder/#arg) rather than ENV, so that these values can be overridden when building the image. I'm ok with ENV though.